# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserChild.cpp
# Commit: d77d76d37d4d
# Full Hash: d77d76d37d4dccf39c12fa81c970088b5388d0f1
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-05-13 08:22:56
# Regressor Bug: 1518919
# File Overlap Count: 1
# Description:
#   Bug 1518919 - Propagate CSS visibility information to descendant documents via IPC call for fission. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30460
# ==============================================================================

diff -r 02b3eb8a1a13 -r d77d76d37d4d dom/ipc/BrowserChild.cpp
--- a/dom/ipc/BrowserChild.cpp	Thu May 09 07:37:03 2019 +0000
+++ b/dom/ipc/BrowserChild.cpp	Mon May 13 01:49:07 2019 +0000
@@ -1265,6 +1265,14 @@
   return IPC_OK();
 }
 
+mozilla::ipc::IPCResult BrowserChild::RecvSetIsUnderHiddenEmbedderElement(
+    const bool& aIsUnderHiddenEmbedderElement) {
+  if (RefPtr<PresShell> presShell = GetTopLevelPresShell()) {
+    presShell->SetIsUnderHiddenEmbedderElement(aIsUnderHiddenEmbedderElement);
+  }
+  return IPC_OK();
+}
+
 bool BrowserChild::UpdateFrame(const RepaintRequest& aRequest) {
   return BrowserChildBase::UpdateFrameHandler(aRequest);
 }